spec/views/post_show_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
```

Comment:
- [ ] Let's ensure to include test for the below scenarios for _post show_ views: 
   * I can see the post's title.
   * I can see who wrote the post.